gensim/topic_coherence/probability_estimation.py

```diff
@@ -79,29 +55,23 @@ def p_boolean_sliding_window(texts, segmented_topics, dictionary, window_size):
     per_topic_postings : Boolean sliding window postings list of all the unique topic ids.
     window_id[0] : Total no of windows
     """
-    top_ids = _ret_top_ids(segmented_topics)
-    window_id = 0  # Each window assigned a window id.
-    per_topic_postings = {}
-    token2id_dict = dictionary.token2id
+    top_ids = unique_ids_from_segments(segmented_topics)
+    if processes <= 1:
+        accumulator = WordOccurrenceAccumulator(top_ids, dictionary)
+    else:
+        accumulator = ParallelWordOccurrenceAccumulator(processes, top_ids, dictionary)
+    logger.info("using %s to estimate probabilities from sliding windows" % accumulator)
```

Comment:
Pass argument as a fnc argument, instead of formatting the string directly with `%`.